pyaedt/modeler/advanced_cad/stackup_3d.py

```diff
@@ -1939,14 +1960,14 @@ class Patch(CommonObject, object):
     frequency : float, None
         The patch frequency, it is used in prediction formulas. If it is None, the patch frequency will be that of the
```

Comment:
```suggestion
        Patch frequency, which is used in prediction formulas. The default is
        ``None``, in which case the patch frequency is that of the
```
Standard wording plus use of present tense.